ui/src/app/views/error-handler/component.html

```diff
@@ -0,0 +1,8 @@
+<div *ngIf="this.errorHandlingService.fiveHundred" class="error-handler">
+  Server Error (500)
+  <clr-icon shape="times" (click)="this.closeFiveHundred()"></clr-icon>
```

Comment:
I'm surprised to see `this` in the template -- I'm used to publishing the controller and using `ctrl`, or publishing fields of the controller to the template's namespace. For example: https://github.com/all-of-us/workbench/search?utf8=%E2%9C%93&q=workspacesLoading&type= . Does that work here -- referencing `errorHandlingService` directly?

Reply:
For some reason that gave me an error. I can give it another shot, but it could be an abstraction by the clarity components?